src/main/java/com/stripe/exception/StripeException.java

```diff
@@ -41,14 +41,14 @@ public StripeException(String message, String requestId, String code, Integer st
    * @return a string representation of the exception.
    */
   @Override
-  public String toString() {
+  public String getMessage() {
     String additionalInfo = "";
     if (code != null) {
       additionalInfo += "; code: " + code;
     }
     if (requestId != null) {
       additionalInfo += "; request-id: " + requestId;
     }
-    return super.toString() + additionalInfo;
+    return super.getMessage() + additionalInfo;
```

Comment:
ah actually question here please.
Do we actually change the behavior here? Such that if we print the error, it will return hashcode like java object?

Reply:
`toString()` calls `getMessage()`, so the behavior shouldn't change here. ErrorProne recommends overriding `getMessage()` instead of `toString()` though: https://errorprone.info/bugpattern/OverrideThrowableToString.